tests/core/htmldataprocessor.js

```diff
@@ -1328,9 +1328,11 @@
 
 	// The `src="&#10;&#106;javascript:..."` is treated as some different protocol in Edge/IE8 and also Chrome on Linux.
 	// IE8 treats it as an URL and opens it which reloads the whole page.
-	// While on Edge the tests passes, the browser prompts with "open as" dialog.
+	// While on Edge and on IE11 the tests passes, the browser prompts with "open as" dialog.
 	// On Chrome Linux it prompts `Open xdg-open?` dialog, which gains focus so other tests requiring focus fails.
-	if ( !( CKEDITOR.env.ie && CKEDITOR.env.version == 8 || CKEDITOR.env.edge ) && !( CKEDITOR.env.chrome && bender.tools.env.linux ) ) {
+	if ( !( CKEDITOR.env.ie && ( CKEDITOR.env.version == 8 || CKEDITOR.env.version == 11 ) ) &&
```

Comment:
Ok, so it happens on latest Windows versions starting from Windows 8. The earliest Windows version which can run IE11 is Windows 7 (here this issue does not occur).
I think there is no point in creating Windows version detection only in order to not ignore this one test in IE11 on Windows 7.

That being said, I am fine with this change.